UXImprovementProcess.md

```diff
@@ -0,0 +1,39 @@
+## UX Improvement Process
+
+```
+zIP: 6
+Layer: Company
+Title: UX Improvement Process
+Author: cosmicbutterglue
+Status: Active
+Type: Process
+Created: 2022-10-02
+License: lol
+```
+## Motivation
+
+UX improvements are hard. Right now, we are gathering UX feedback from many different places (Twitter, contribution games, Bitcointalk, etc.). The problem: The UX research we gather is not reproducible. We may get many different opinions, but are unable to identify where exactly the user is encountering issues (and where they aren't). This results in improving UX in the dark. We may identify pain points and resolve them, but we may also identify them wrongly and make the user experience worse by accident. This results in a lot of wasted time for developers as well as unhappy users, which are both scenarios we should try to avoid as best we can.
+
+## Develop a Streamlined UX Improvement Process
+
+To avoid wasting time on developing UX solutions which may not be needed, or worse, make UX more complicated, we should develop a streamlined UX improvement process. This could look something like this:
+
+### 0. Form a pool of UX Testers
+
+Establish a pool of UX Testers comprised of advanced users and noobs. Ideally both groups are made up of 10-20 people or more. For every round of testing, Testers should receive small bounties to motivate participation.
```

Comment:
I would emphasize on focusing on noob users, because that is what the product is meant to target.

Reply:
Agreed and updated!